spring-boot/admin/src/main/java/eu/coatrack/admin/GitHubSecurityConfig.java

```diff
@@ -57,13 +52,6 @@ protected void configure(HttpSecurity http) throws Exception {
                 .logout().logoutUrl("/logout").logoutSuccessUrl("/").deleteCookies("XSRF-TOKEN", "auth_code", "JSESSIONID").invalidateHttpSession(true).clearAuthentication(true).permitAll()
                 .and()
                 //.csrf().csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse());
-                .csrf().disable();
+                .csrf().disable().oauth2Login();
     }
-
-    @Bean
-    @Primary
-    public ResourceServerProperties resource() {
-        return new ResourceServerProperties();
```

Comment:
To be double-checked by @BrunoSilvaATB: do we still need this for the external API?

Reply:
I tested and it worked

Reply:
The production code already has such a bean. Leaving resourceServerProperties in the production code would cause ambiguous beans which makes Spring Boot fail to start. The test code on the other hand requires such a bean which is why moved it to there.

Reply:
The problem is that this ResourceServerProperties works with other functionality of CoatRack, so this can't disappear.
This functionality is about reading/extracting the token from the request Header and use it to log in to CoatRack, so that our partners can call our public api with a token in the header.
Regarding failing to start and "ambitious beans", maybe you can show that to me on our TelCo today, so that I get a better idea